src/pages/NgramViewer.js

```diff
@@ -46,13 +101,68 @@ function NgramViewer() {
                 <a href="https://en.wikipedia.org/wiki/N-gram">Wikipedia</a>
```

Comment:
you can add a `target="_blank" rel="noopener noreferrer"` here so it opens to a new tab. If users comes back to our site they'll have to relogin

Reply:
truuuuu

Reply:
THANK U!!

Reply:
(added)